docs/api.md

```diff
@@ -2143,6 +2151,39 @@ Identifies what kind of target this is. Can be `"page"`, `"service_worker"`, or
 #### target.url()
 - returns: <[string]>
 
+### class: Coverage
+
+#### coverage.startJSCoverage()
```

Comment:
Add note about coverage not surviving navigations. I might try starting coverage at the start of my tests, ending at the end of my tests, and reporting sections that are not used. We use this pattern ourselves.